src/Selenium2Library/keywords/_element.py

```diff
@@ -297,6 +297,16 @@ def get_horizontal_position(self, locator):
             raise AssertionError("Could not determine position for '%s'" % (locator))
         return element.location['x']
 
+    def get_element_size(self, locator):
+        """Returns width and height of element identified by `locator`.
+
+        The element width and height is returned.
+        Fails if a matching element is not found.
+        """
+        element = self._element_find(locator, True, True)
+
+        return element.size['width'], element.size['height']
```

Comment:
It might be a good idea to return a [namedtuple](https://docs.python.org/2/library/collections.html#collections.namedtuple) instead of a normal tuple. It would allow this kind of usage:

    ${size} =    Get Element Size    ${locator}
    Should Be Equal    ${size.height}    42
    Should Be Equal    ${size.width}    0

Also, the empty line above the `return` statement is unnecessary.